front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -83,13 +85,14 @@ export const formatSuggestedViasToRowVias = (
     const formattedDeparture: TimeExtraDays | undefined = departureTime
       ? { time: departureTime }
       : undefined;
+    const { receptionSignal: _opReceptionSignal, ...filteredOp } = op;
     return {
-      ...op,
+      ...filteredOp,
       isMarginValid,
       arrival: formattedArrival,
       departure: formattedDeparture,
-      onStopSignal: onStopSignal || false,
-      name: name || t('waypoint', { id: op.opId }),
+      onStopSignal: receptionSignalToOnStopSignal(receptionSignal),
```

Comment:
Why do we need to keep `onStopSignal` here? Shouldn't this be changed to a receptionSignal too?

Reply:
There is still only a check box in the form, and its meaning is still "onStopSignal".
And the plan is to keep this checkbox and pop another one when on stop signal to trigger short slip distance.

So the form part keeps its semantic and the app state migrates to "receptionSignal", with a translation in between.